browser/templates/browser/person.html

```diff
@@ -24,6 +24,10 @@
                 aria-hidden="true"
                 href="{% url "split-person" person.id %}">
             </a>
+            <a class="btn btn-lg fa fa-plus"
+                aria-hidden="true"
+                href="{% url "add-investigator" person.id %}">
```

Comment:
please at a title so there is a hover description. and while you're at it, please add titles for the other buttons as well.